src/Guidelist.jsx

```diff
@@ -93,5 +93,9 @@ GuideList.propTypes = {
     slug: string,
     url: string,
   }).isRequired,
-  promoted_guide: shape().isRequired,
+  promoted_guide: shape(),
```

Comment:
Do you know what this object looks like? I don't think I came across a promoted guide, so couldn't see an example. Not major and still happy for it to be merged, just would be nice.